extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -59,6 +59,8 @@ tags: {  # <amp-state> with json child
     name: "id"
     mandatory: true
   }
+  # <amp-bind>
```

Comment:
That's what the example above does though good point that this is unintuitive. This would also cause verification in dev mode to fire an error. Hmm...